paimon-core/src/main/java/org/apache/paimon/table/system/SnapshotsTable.java

```diff
@@ -260,7 +267,12 @@ private InternalRow toRow(Snapshot snapshot, FileStoreTable dataTable) {
                     snapshot.changelogRecordCount(),
                     plan.files(FileKind.ADD).size(),
                     plan.files(FileKind.DELETE).size(),
-                    snapshot.watermark());
+                    snapshot.watermark(),
+                    statsFileHandler
+                            .readStats(snapshot)
+                            .map(Statistics::toString)
```

Comment:
Maybe it is not a good idea to make string for a whole statistics... To large.

Maybe we can have some other system table to show full statistics?

Reply:
+1.